package.nls.json

```diff
@@ -262,7 +262,7 @@
     "AWS.sam.debugger.missingField": "Missing required field \"{0}\" in debug config",
     "AWS.sam.debugger.resourceNotAFunction": "Template Resource {0} in Template file {1} needs to be of type {2}",
     "AWS.sam.debugger.unsupportedRuntime": "Runtime for Template Resource {0} in Template file {1} is either undefined or unsupported.",
-    "AWS.sam.debugger.useExistingConfig": "The Toolkit has detected an existing legacy configuration for this function handler. Would you like it added to your Debug Configuration?",
+    "AWS.sam.debugger.useExistingConfig": "AWS Toolkit detected an existing legacy configuration for this function. Add to launch.json?",
```

Comment:
When this situation is detected, there are three courses of action that the Toolkit can take for the user:
* create a config that is complemented with existing legacy config details
* create a config that ignores the existing legacy config details
* back out and do nothing

This change removes the second action in the list. Would it be reasonable to provide it as an option for users?

Reply:
Does not seem worth the complication to me (and that was supported by the confusing nature of the old dialog). But I'm willing to make any concrete suggestion you have, in terms of dialog buttons and verbiage

Reply:
It doesn't make sense to leave out the second option but leaving out the third option would be reasonable (we don't prompt the user to confirm whether or not they wanted to create a launch config from something without a legacy config). We'd have to craft the message to reflect the "Cancel" option as viable though. Thoughts?